static/js/search.js

```diff
@@ -2,9 +2,33 @@ var search = (function () {
 
 var exports = {};
 
+function ValidURL(str)
```

Comment:
* In javascript, we generally use function names spelled e.g. `valid_url`.
* Generally we put the open curly brace on the same line as the `function` statement.